Source/Painting/SvgColourConverter.cs

```diff
@@ -165,6 +163,10 @@ public override object ConvertFrom(System.ComponentModel.ITypeDescriptorContext
                         // have to prevent this and ignore the color instead (see #342) 
                         return SvgColourServer.NotSet;
                     }
+
+                    var index = colour.LastIndexOf("grey", StringComparison.InvariantCultureIgnoreCase);

```

Comment:
If you only support "LightGrey", use `ColorTranslator`, but major browsers support "Grey" group.

Reply:
Looks like the `ColourServer` is a `ColorServer` in reality 😛